src/Internal/Plutip/Server.purs

```diff
@@ -473,10 +477,18 @@ startPlutipCluster
   -> Aff (PrivatePaymentKey /\ ClusterStartupParameters)
 startPlutipCluster cfg keysToGenerate = do
   let
+    startUpRequest = case cfg.clusterConfig of
```

Comment:
With my suggestions below, this would be changed to add `keysToGenerate` and the hardcoded `epochSize` to `cfg.clusterConfig` to form a `ClusterStartupRequest` record

Reply:
[38cdff7](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1334/commits/38cdff7c12fa504e3990faff8491a5bab94d9148)
Merge state
Add more commits by pushing to the [brad/update-plutipConfig](https://github.com/Plutonomicon/cardano-transaction-lib/tree/brad/update-plutipConfig) branch on [Plutonomicon/cardano-transaction-lib](https://github.com/Plutonomicon/cardano-transaction-lib).

This branch has not been deployed
No deployments
Review required
At least 1 approving review is required by reviewers with write access. 
8 pending reviewers
Some checks haven’t completed yet
1 queued, 1 pending, and 1 successful checks
@github-actions
Check Markdown links / markdown-link-check (push) Queued — Waiting to run this check...
[Details](https://github.com/Plutonomicon/cardano-transaction-lib/actions/runs/3734226930/jobs/6336019145)
@hercules-ci
ci/hercules/onPush/default Pending — Waiting for agent(s)
[Details](https://hercules-ci.com/github/Plutonomicon/cardano-transaction-lib/jobs/3636)
@hercules-ci
ci/hercules/evaluation — onPush jobs created.
[Details](https://hercules-ci.com/github/Plutonomicon/cardano-transaction-lib/jobs/3635)
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1334/conflicts) or the  to resolve conflicts.
Conflicting files
src/Internal/Plutip/Types.purs
 You can also [open this in GitHub Desktop](x-github-client://openRepo/https://github.com/Plutonomicon/cardano-transaction-lib?branch=brad%2Fupdate-plutipConfig) or view .
@Bradley-Heather

 
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Leave a comment
No file chosen
Attach files by dragging & dropping, selecting or pasting them.
Styling with Markdown is supported
Remember, contributions to this repository should follow our [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
 ProTip! Add comments to specific lines under [Files changed](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1334/files).
Reviewers
@[rynoV](https://github.com/rynoV)
rynoV
@[jy14898](https://github.com/jy14898)
jy14898
@[amirmrad](https://github.com/amirmrad)
amirmrad
@[errfrom](https://github.com/errfrom)
errfrom
@[kirill-havryliuk](https://github.com/kirill-havryliuk)
kirill-havryliuk
@[Luis-omega](https://github.com/Luis-omega)
Luis-omega
@[uhbif19](https://github.com/uhbif19)
uhbif19
@[zmrocze](https://github.com/zmrocze)
zmrocze
@[klntsky](https://github.com/klntsky)
klntsky
At least 1 approving review is required to merge this pull request.

Still in progress? 
Assignees
No one—
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet

Notifications
Customize
You’re receiving notifications because you authored the thread.
3 participants
@Bradley-Heather
@klntsky
@rynoV